samples/doc-snippets/src/main/resources/static/index.html

```diff
@@ -0,0 +1,44 @@
+<html>
+    <head>
+        <title>Static content sample</title>
+        <link rel="stylesheet" href="/static/style.css"/>
```

Comment:
would this also work with relative `href="style.css"` ?

Reply:
I added a route to publish the index.html file at `/index.html`, relative paths are from where the main page comes, so `style.css` alone would only work if the page itself is only made available under `/static/index.html`

Reply:
makes sense, that is good